ospf6d/ospf6_neighbor.c

```diff
@@ -1075,9 +1076,12 @@ DEFUN (show_ipv6_ospf6_neighbor_one,
 
 	for (ALL_LIST_ELEMENTS_RO(ospf6->area_list, i, oa))
 		for (ALL_LIST_ELEMENTS_RO(oa->if_list, j, oi))
-			for (ALL_LIST_ELEMENTS_RO(oi->neighbor_list, k, on))
-				(*showfunc)(vty, on, json, uj);
-
+			for (ALL_LIST_ELEMENTS_RO(oi->neighbor_list, k, on)) {
```

Comment:
please compare router_id to on->router_id

Reply:
Done